src/Infrastructure.VS.UnitTests/ErrorListHelperTests.cs

```diff
@@ -60,8 +75,8 @@ public void TryGetRuleIdFromSelectedRow_SingleSonarIssue_ErrorCodeReturned(strin
                 { StandardTableKeyNames.ErrorCode, fullRuleKey }
             });
 
-            var mockErrorList = CreateErrorList(issueHandle);
-            var serviceProvider = CreateServiceProvider(mockErrorList);
+            var errorList = CreateErrorList(issueHandle);
```

Comment:
I renamed all the "mockErrorList" to errorList as the method doesnt actually return a mock so the naming confused me.